react/src/js/react/App.jsx

```diff
@@ -32,6 +33,7 @@ export default class App extends React.Component {
                         <Route exact path="/feed" component={PostFeed} />
                         <Route path="/register" component={RegisterView} />
                         <Route path="/profile/:id" component={ProfileView} />
+                        <Route path="/discover/:id" component={PostView} />
```

Comment:
I think this path would be better off being /post/:id instead of /discover/, because we're also going to have posts on other routes as well